src/README.md

```diff
@@ -101,3 +101,19 @@
 - ID: **EOF1C0017** legacy contact > CREATE[2] > legacy init code > nonsense
 - ID: **EOF1C0018** legacy contact > CREATE[2] > eof1 init code > nonsense (fail)
 - ID: **EOF1C0019** legacy contact > CREATE[2] > nonsense init code (fail)
+
+
+## Warm COINBASE EIP-3651 
+
+- ID: **WRMCB0001** check EXTCODESIZE gas cost on coinbase
```

Comment:
the same key should also be mentioned in the test file in comments or in info section comment field. so we can find the test.